### PR TITLE
[11.0] account_asset_disposal: Migration to 11.0

### PR DESCRIPTION
**Pull request by pedrobaeza**
* Standard procedure
* Remove migration script
* README by fragments
* Update headers
* Remove coding

cc @Tecnativa

### CODE REVIEW

File: account_asset_disposal/README.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--financial--tools-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-financial-tools/tree/12.0/account_asset_disposal
```
Comment (bodedra) [inline account_asset_disposal/README.rst]: 11.0 ?

Comment (pedrobaeza) [reply account_asset_disposal/README.rst]: Right, I ran the script with incorrect parameter. Changing...

File: account_asset_disposal/README.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    :alt: OCA/account-financial-tools
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-financial-tools-12-0/account-financial-tools-12-0-account_asset_disposal
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/92/12.0
```
Comment (bodedra) [inline account_asset_disposal/README.rst]: @pedrobaeza seems readme script run for 12.0 ;-) It should be 11.0

Comment (pedrobaeza) [reply account_asset_disposal/README.rst]: Done!

File: account_asset_disposal/wizards/account_asset_disposal_wizard_view.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<openerp>
```
Comment (bodedra) [inline account_asset_disposal/wizards/account_asset_disposal_wizard_view.xml]: s/openerp /odoo

File: account_asset_disposal/wizards/account_asset_disposal_wizard_view.xml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+<data>
```
Comment (bodedra) [inline account_asset_disposal/wizards/account_asset_disposal_wizard_view.xml]: IMO We can remove data tag.

File: account_asset_disposal/wizards/account_asset_disposal_wizard_view.xml (hunk 3 above)
Comment (pedrobaeza) [reply account_asset_disposal/wizards/account_asset_disposal_wizard_view.xml]: Old remnants from initial 8.0 version. Changed!

File: account_asset_disposal/wizards/account_asset_disposal_wizard_view.xml (hunk 4 above)
Comment (pedrobaeza) [reply account_asset_disposal/wizards/account_asset_disposal_wizard_view.xml]: Yes, indeed.

**Review by bodedra — APPROVED** [review]
Code review :+1:

**Comment by pedrobaeza** [comment]
@luc-demeyer `account_asset_management` is still defined as incompatible with `account_asset`, so this is failing, but I remember you said that they are no longer incompatible. Is that true?

**Comment by JordiBForgeFlow** [comment]
@eficent

**Review by AaronHForgeFlow — COMMENTED** [review]
:+1: Functional test good. @pedrobaeza Do you have a clue why CI is red?

File: account_asset_disposal/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    "summary": "Makes asset close account move automatically",
+    "version": "11.0.1.0.0",
+    "category": "Accounting & Finance",
+    "website": "http://github.com/OCA/account-financial-tools",
+    "author": "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": [
+        "account_asset",
```
Comment (Tardo) [inline account_asset_disposal/__manifest__.py:17]: This module appears to be incompatible with 'account_asset_management' that is installed on travis.

**Comment by pedrobaeza** [comment]
I have put isolated travis builds for this module, as there's no way to compatibilize both modules.

File: account_asset_disposal/__manifest__.py (hunk 5 above)
Comment (pedrobaeza) [reply account_asset_disposal/__manifest__.py:17]: Solved

**Comment by sbidoul** [comment]
@pedrobaeza don't forget it's now easy to bump version (patch|minor|major after merge command)

**Comment by pedrobaeza** [comment]
Thanks for the reminder, @sbidoul :smiley: . In this case, being a module migration, it doesn't apply.

**Pull request closed without merging**